linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_screensaver_mode_blank/rule.yml

```diff
@@ -1,10 +1,28 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8
+prodtype: fedora,ol7,ol8,rhel7,rhel8,sle12,sle15
 
 title: 'Implement Blank Screensaver'
 
 description: |-
+    {{# The SLE12 and SLE15 STIG unfortunately requires per-user settings, #}}
+    {{# not a globally locked one, so we don't require but only suggest the latter. #}}
+    {{% if product == "sle15" }}
```

Comment:
The block:
    {{% if product in ["sle12","sle15"]%}}
    On SUSE users should set the screensaver to use publicly viewable images or blank screen
    by doing the following to configure the system.  Find the
    Settings menu and then navigate to the Background selection section:

should have remained,
So we have stuff that applies to both sl15 and sle12, with in that there info that is specific to sl15 and other info that is sle12 specific.

Reply:
Fixed